### PR TITLE
Set status of passing test to passing

### PR DESCRIPTION
**Pull request by orangejulius**
It was set to fail, which we don't want to do anymore.

### CODE REVIEW

File: test_cases/suggest_coarse.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "tests": [
     {
       "id": 1,
-      "status": "fail",
+      "status": "pass",
       "comment": "Currently fails due to the fact that suggest doesn't boost more popular records to the top of search results, so `Brooklyn` appears below other less well-known records with `Brooklyn` in their name.",
```
Comment (hkrishna) [inline test_cases/suggest_coarse.json]: do you want to update the comment as well?

Comment (orangejulius) [inline test_cases/suggest_coarse.json]: Oh, yeah, of course. I've updated the PR with that comment removed

**Comment by hkrishna** [comment]
:+1:

**Comment by orangejulius** [comment]
Merged on the CLI

**Pull request closed without merging**